packages/middleware-retry/src/retryMiddleware.ts

```diff
@@ -20,6 +21,7 @@ import { INVOCATION_ID_HEADER, REQUEST_HEADER } from "@smithy/util-retry";
 import { v4 } from "uuid";
 
 import { RetryResolvedConfig } from "./configurations";
+import { isStreamingPayload } from "./isStreamingPayload/isStreamingPayload";
```

Comment:
Rename `isStreamingPayload/isStreamingPayload.ts` to `isStreamingPayload/index.ts` so that it can be directly imported?

Reply:
I usually open files by typing in their name, so naming things index makes it hard for me to find